config/forms/2021_2022.json

```diff
@@ -181,7 +181,7 @@
                 "propcode": {
                   "check_answer_label": "Property reference",
                   "header": "What is the property reference?",
-                  "hint_text": "This is how you refer to the property in your own systems",
+                  "hint_text": "This is how you usually refer to this property on your own systems.",
```

Comment:
On/in feels like a fun grammatical rabbit hole :)

Reply:
You know it!